widgets/pagination.js

```diff
@@ -1,6 +1,8 @@
 var React = require('react');
 
 var utils = require('../lib/utils.js');
+var autoHide = require('../decorators/autoHide');
+var Pagination = autoHide(require('../components/Pagination/Pagination.js'));
```

Comment:
> Autocompletion?

Yes but I am more concerned about reaching the 80 line limit with a single variable name! we're good